bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java

```diff
@@ -1009,6 +1009,9 @@ public void scanEntryLog(long entryLogId, EntryLogScanner scanner) throws IOExce
                     continue;
                 }
                 // read the entry
+                if (entrySize == 0) {
```

Comment:
Why aren't you doing the same as at line 1017?
We should log and return

Reply:
Yes, it's probably better to log it when it happens